.gitignore

```diff
@@ -1,14 +1,38 @@
-*.iml
+/unbundled-build
```

Comment:
this looks a lot more complex than the arch ones:

https://github.com/googlesamples/android-architecture-components/blob/master/.gitignore
https://github.com/googlesamples/android-architecture-components/blob/master/BasicSample/.gitignore

Reply:
Ah.. I took it from androidx. I guess we can just ignore the whole .idea folder in the samples